docs-source/content/faq/oci-lb.md

```diff
@@ -9,19 +9,19 @@ If you are running your Kubernetes cluster on Oracle Container Engine
 for Kubernetes (commonly known as OKE), you can have OCI automatically
 provision load balancers for you by creating a `Service` of type
 `LoadBalancer` instead of (or in addition to) installing an
-ingress controller like Traefik or Voyager.
+Ingress controller like Traefik or Voyager.
 
 OKE Kubernetes worker nodes normally do not have public IP addresses.
 This means that the `NodePort` services created by the operator are
 not usable, because they would expose ports on the worker node's private
 IP addresses only, which are not reachable from outside the cluster.  
-Instead, you can use an OCI Load Balancer to provide access
+Instead, you can use an OCI load balancer to provide access
 to services running in OKE.
 
 {{% notice note %}}
-It is also possible, if desirable, to have an OCI Load Balancer route
-traffic to an ingress controller running inside the Kubernetes cluster
-and have that ingress controller in turn route traffic to services in the
+It is also possible, if desirable, to have an OCI load balancer route
```

Comment:
we need to update this -- our recommendation should be to use both an oci lb which sends traffic to the ingress controller and then handle routing on the ingress.  this is fine for now - but we should probably make a jira so we (i) don't forget to update this